Applications/CLI/ogs.cpp

```diff
@@ -96,6 +119,22 @@ int main(int argc, char *argv[])
 
 	output_file.close();
 
+	INFO("Elapsed running time in total computation %g s", run_timer.elapsed());
```

Comment:
suggest making it short:
"Elapsed running time" -> "Elapsed time"
"Elapsed CPU time" -> "CPU time"